system/jlib/jsocket.cpp

```diff
@@ -2094,8 +2138,10 @@ size32_t CSocket::writetms(void const* buf, size32_t minSize, size32_t size, uns
                     errclose();
                     err = JSOCKERR_broken_pipe;
                 }
-                if ((err == JSE_WOULDBLOCK) && nonblocking)
+                if (err == JSE_WOULDBLOCK)
```

Comment:
Do we also check if err == EAGAIN here ?

Reply:
yes, I think should. I'll change it.